indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
this usecase does not look like something that needs special handling here. You can either write and use a InputRowParser implementation that handles BytesWritable or extend your choice of InputFormat so that it returns ByteBuffer .